src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -379,13 +376,7 @@ private void setupPubSub(@NonNull Config cfg, NotificationUtil notificationUtil)
         }
 
         try {
-            ElasticExportSubscription.subscribeElasticExport(projectId, elasticExportSubscriptionId);
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-
-        try {
-            UpdateWorkflowStatusSubscription.subscribeUpdateWorkflow(projectId, updateWorkflowSubscriptionId);
+            DSMtasksSubscription.subscribeDSMtasks(projectId, DSMtasksSubscriptionId);
```

Comment:
Too verbose to have whole pubsub receiver/subscriber setting up from line 330 to 369 in `DSMServer`, I would separate it in different class or in already pubsub existing class, separating database connection/operations on line 336 in `DAO` class would be better as well.